site/contents/write_ups/2021/03-11-21_writeup.md

```diff
@@ -0,0 +1,109 @@
+"Title  II  establishes  a  list  of  prohibited  AI" "Title III contains specific rules for AI systems that create a high risk to the health and safety or  fundamental  rights  of  natural  persons." "Title IV concerns certain AI systems to take account of the specific risks of manipulation they pose."
+
+- [Italy AI Strategy Report](https://knowledge4policy.ec.europa.eu/ai-watch/italy-ai-strategy-report_en) (2021).
+- [Policy initiatives for Italy](https://oecd.ai/en/dashboards/policy-initiatives?conceptUris=http:%2F%2Fkim.oecd.org%2FTaxonomy%2FGeographicalAreas%23Italy).
+- [Italian AI white paper 2018](https://ia.italia.it/assets/whitepaper.pdf).
+
+In 2018 Italy adopted a national strategy on AI, following EU guidelines and the example of other European countries. The Italian government will release a new plan in 2022.
+
+- [Strategia nazionale per l'intelligenza artificiale - Contesto](https://www.mise.gov.it/index.php/it/strategia-intelligenza-artificiale/contesto) (in Italian).
+- [Draghi’s strategy for Italy’s AI - decode39](https://decode39.com/2162/italy-ai-strategy-draghi/).
+
+"Operating within the framework of the coordinated plan on European AI published in December 2018, Mr Draghi’s government strategy aims to increase public funds for AI research".
+
+## The report states that they want to build the “most trusted and pro-innovation system for AI governance in the world” - can both these things be true?
+
+Ideally you want to find as many bugs as you can. You cannot find all of them, as you have limited time. So at what point do you start just calling them features? What sort of prescedent does that set? Could you balance reducing the bureacracy and innovative drive? Probably not consistently. But if you consider the pharamceutical industry's approach to developing various COVID 19 vaccines, private companies were able to maintain the strict standards of their respective ethical bodies but with a streamlined process (in their case, producing batches for the next stage of the trial before they had moved along to that trial, risking wasting that entire batch should it fail the stage of the trial).
+
+We recalled studies where different people were given a software task. They found that diverse teams reduce errors because people with different backgrounds made different errors. 
+
+You will not have innovation in the long term if people do not trust your product - they will just pull the plug. A lot of the AI being commerical ties into what public thinks about it. If people don't like a product, they won't use it. Public opinion almost forms an accidental ethics board in a more consumerist, capitalist society. 
+
+Consumerism should not be the only line of defense to unethical practise. We need formal ethical boards to block unethical developments as well as guide innovators. Scientists should innovate in tramlines rather than in an open space.
```

Comment:
```suggestion
That being said, consumerism should not be the only line of defense to unethical practice, especially sicne the public can only react to what they have been informed about. Undoubtedly much of our data is used in ways we do not understand, and therefore cannot protest against. Because of that, we all felt that we needed formal ethical boards to block unethical developments as well as to guide innovators. Someone suggested this as the idea that scientists should innovate in tramlines rather than in an open space.
```